index.html

```diff
@@ -339,28 +282,23 @@ <h3>
           </li>
           <li>Let |promise:Promise| be [=a new promise=].
           </li>
-          <li>Run the following steps <a>in parallel</a>:
+          <li>
+            <a>Queue a global task</a> on the <a>screen wake lock task
+            source</a> given |document|'s <a>relevant global object</a> to run
+            these steps:
             <ol>
-              <li>Let |state:PermissionState| be the result of <a>requesting
-              permission to use</a> "`screen-wake-lock`".
-              </li>
-              <li data-tests="wakelock-request-denied.https.html">If |state| is
-              {{PermissionState/"denied"}}, then:
+              <li>If |document|'s [=Document/visibility state=] is "hidden", then:
```

Comment:
Something went wrong with the merge in these steps too: all the main substeps within the "Queue a global task" step have been duplicated inside the _If document.[[ActiveLocks]]["screen"] is empty_ substep (see the [Preview](https://pr-preview.s3.amazonaws.com/w3c/screen-wake-lock/pull/326.html#the-request-method) link).